Makefile

```diff
@@ -382,6 +382,7 @@ manifest:
 	@echo "===> Generating dev manifest for Antrea <==="
 	$(CURDIR)/hack/generate-standard-manifests.sh --mode dev --out build/yamls
 	$(CURDIR)/hack/generate-manifest-windows.sh --mode dev > build/yamls/antrea-windows.yml
+	$(CURDIR)/hack/generate-manifest-windows.sh --mode dev --containerd > build/yamls/antrea-windows-containerd.yml
```

Comment:
So the current yaml in code base was generated manually?

Reply:
In release mode, we generate both docker and containerd YAMLs from /hack/release/prepare-assets.sh. However, I forgot to add the containerd generation in make manifest, which caused the github action to skip checking if it is up-to-date.